README.rst

```diff
@@ -14,6 +14,21 @@ and latest releases of the standard.
 
 Documentation
 -------------
-
 For more information visit `the documentation page
 <http://emva1288.readthedocs.io/en/latest/>`__
+
+Camera Testing : Pytest
+-------------
+Install Requirements : *pip install -r requirements.txt*
```

Comment:
Instead of adding test dependencies to requirements.txt, it should be split into 2, 1 for main requirements (requirements.txt) and other for test requirements called requirements-test.txt. This way we can have option in setup.py to install test requirements only if required.

Reply:
Good point. I will create a separate requirements-test.txt with only test related dependancies  as you recommend.